.github/workflows/sca_scan.yaml

```diff
@@ -0,0 +1,32 @@
+name: Snyk Software Composition Analysis Scan
+# This git workflow leverages Snyk actions to perform a Software Composition 
+# Analysis scan on our Opensource libraries upon Pull Requests to Main & 
+# Develop branches.  We use this as a control to prevent vulnerable packages 
+# from being introduced into the codebase. 
+on:
+  pull_request:
+    branches: 
+      - master
+      - develop
+jobs:
+  Snyk_SCA_Scan:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+      - uses: actions/checkout@master
+      - name: Setting up Node
+      #- name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Installing snyk-delta and dependencies
+        run: npm i -g snyk-delta
+      - uses: snyk/actions/setup@master
```

Comment:
Do we want to reference master or a particular version?

Reply:
Master